decide/models/enums.py

```diff
@@ -9,15 +12,37 @@ class Currency(Enum):
 
 class Bank(Enum):
     ACCESS = "044"
+    ALAT = "035A"
     CIB = "818147"
+    ECOBANK = "050"
     FBN = "011"
+    FCMB = "214"
+    FIDELITY = "070"
+    GLOBUS = "00103"
     GTB = "058"
     HSBC = "818039"
-    UBA = "033"
+    KEYSTONE = "082"
+    KUDA = "50211"
     MBS = "041"
     MPESA = "404001"
+    PROVIDUS ="101"
+    POLARIS = "076"
+    STANBIC = "221"
+    STERLING = "232"
+    UBA = "033"
+    UNITY = "215"
+    UNION = "032"
     ZENITH = "057"
 
+    @classmethod
+    def get_bank_list(cls):
```

Comment:
Shouldn't this be added to changelog file?

Reply:
@aderopoa thanks for your question
The Changelog is automatically generated(updated) with the commit messages, the commit messages already references that a new method was added, will also add to documentation. I hope my answer is on point?

Reply:
The auto generation happens after the PR is merged